### ISSUE TITLE
htmlsanity incompatible with docutils 0.17

### ISSUE THREAD

**Issue by lpirl** [mosra/m.css#194]
The plugin `m.htmlsanity` seems to be incompatible with doxygen >= 0.17.
Upon content generation the following error occurs:
```
AttributeError: 'SaneHtmlTranslator' object has no attribute 'object_image_types'
```

**Comment by mosra** [comment]
Should be fixed in f9ec8ea832313d37789ed078f5def98908b4be08.

**Comment by lpirl** [comment]
Phew that fix came almost before I could file the issue. Scary… in a good way. Thanks. :)
